src/app/features/de-duplication/bulk-merge-records/merge-fields/merge-fields.component.ts

```diff
@@ -0,0 +1,172 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { AbstractControl } from "@angular/forms";
+import { Entity } from "app/core/entity/model/entity";
+import { EntityFieldViewComponent } from "app/core/common-components/entity-field-view/entity-field-view.component";
+import { EntityFieldEditComponent } from "app/core/common-components/entity-field-edit/entity-field-edit.component";
+import { MatCheckboxModule } from "@angular/material/checkbox";
+import { MatRadioModule } from "@angular/material/radio";
+import { CommonModule } from "@angular/common";
+import { FormFieldConfig } from "app/core/common-components/entity-form/FormConfig";
+
+@Component({
+  selector: "app-merge-fields",
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatCheckboxModule,
+    MatRadioModule,
+    EntityFieldViewComponent,
+    EntityFieldEditComponent,
+  ],
+  templateUrl: "./merge-fields.component.html",
+  styleUrls: ["./merge-fields.component.scss"],
+})
+export class MergeFieldsComponent implements OnInit {
+  @Input() field!: FormFieldConfig;
+  @Input() entities!: Entity[];
+  @Input() control!: AbstractControl;
+
+  existingSelected: boolean[] = [];
+  isDisabled: boolean[] = [];
+  allowsMultiValue: boolean = false;
+
+  ngOnInit() {
+    this.initializeFieldState();
+    this.control?.valueChanges.subscribe((newValue) => {
+      this.updateSelectedStatus(newValue);
+      this.control?.patchValue(newValue, { emitEvent: false });
```

Comment:
does this really make sense? We are waiting for the control value to change, and then patch that same new value to the same control?